### ISSUE TITLE
Training stopping criteria: fixed epochs ?

### ISSUE THREAD

**Issue by Aakriti05** [openai/weak-to-strong#36]
I wanted to know if fixed training epoch has been used for training and as a stopping criteria ?
Generally a validation set is used. I wanted to know what criteria has been used to report the final results and why ? 

Thanks
